### ISSUE TITLE
GitHub Actions failing with below error 

### ISSUE THREAD

**Issue by sravula84** [google-github-actions/setup-gcloud#621]
### TL;DR

0s
Post job cleanup.
/usr/local/bin/docker exec  fde52bc577f3ffa47f08dde1b995f8e5048efb8c59b6c54fd549fcef939685f2 sh -c "cat /etc/*release | grep ^ID"
Error relocating /__e/node16_alpine/bin/node: getentropy: symbol not found

### Expected behavior

Post job cleanup.
/usr/local/bin/docker exec  aabbc6191d9c0def99146456c1014e887b58027e7f566e17c10183f089ef36b2 sh -c "cat /etc/*release | grep ^ID"
Removed exported credentials at "/__w/web-authentication/web-authentication/gha-creds-e378a8e9a786ba2b.json".

### Action YAML

```yaml
- name: Authenticate to Google Cloud
        uses: google-github-actions/auth@v1
        with:
          credentials_json: '${{xxxxxx.xxxxx }}'

 - name: 'Set up Cloud SDK'
        uses: 'google-github-actions/setup-gcloud@v1'

 - name: Build Image with kaniko executor
        run: |
          echo ${{ github.workspace }}
          gcloud auth activate-service-account ${{ vars.xxxxx }} \
          --key-file=${xxxx} \
          --project=${xxxxx}

 /kaniko/executor -f `pwd`/Dockerfile -c `pwd` --build-arg version=$CICD_ARTIFACT_VERSION \
                              --build-arg artifactory_publish_flag=$artifactory_publish_flag \
                              --build-arg apollo_key=${{env.apollo_key}} \
                              --build-arg graph_name=${{env.graph_name}} \
                              --build-arg cicd_schema_pr_validate=${{env.cicd_schema_pr_validate}} \
                              --build-arg artifactory_user=${ARTIFACTORY_USERNAME} \
                              --build-arg artifactory_password=${ARTIFACTORY_PASSWORD} \
                              --build-arg service_name=${{github.event.repository.name}} \
                              --no-push
```

### Additional information

this is happening only for few repos work flow, most of the repos working fine with out any issues, but some repo failing with the following error 

looks like unable to remove the credentials file. we are running this on self hosted runner

**Comment by sethvargo** [comment]
Hi there - could you please provide the [debug output](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging) for the complete GitHub Actions workflow run?

**Comment by sravula84** [comment]
[1_CI  kaniko.txt](https://github.com/google-github-actions/setup-gcloud/files/10947032/1_CI.kaniko.txt)
[actions-logs2.txt](https://github.com/google-github-actions/setup-gcloud/files/10947118/actions-logs2.txt)
[action-logs1.txt](https://github.com/google-github-actions/setup-gcloud/files/10947119/action-logs1.txt)

**Comment by sethvargo** [comment]
Hi @sravula84 

I'm seeing a few errors like:

```text
 ---> System.UnauthorizedAccessException: Access to the path '/runner/_work/_temp/_github_home/.config/gcloud/legacy_credentials/github-action@prosper-production.iam.gserviceaccount.com' is denied.
```

That make me think this is a file permissions issue or a self-hosted runner misconfiguration. Since there are many ways to configure self-hosted runners, our ability to debug and assist is extremely limited.

You could try setting `cleanup_credentials` to false in `google-github-actions/auth`, but **warning, this will leave authentication credentials on disk for future runs!** Self-hosted runners are the reason why the cleanup step exists in the first place.

**Comment by sravula84** [comment]
ok Thank you 

do I need to set the cleanup_credentials to false in the below code ?

- name: Authenticate to Google Cloud
        uses: google-github-actions/auth@v1
        with:
          credentials_json: '${{ secrets.CICD_GCP_SERVICE_ACCOUNT_KEY }}'
          cleanup_credentials: false

**Comment by sethvargo** [comment]
Yes, but keep in mind that credentials will exist on the runner even after the job finishes, which could be used by future jobs.

**Comment by sravula84** [comment]
we are running this runner in Kubernetes pod , so when the job finish pod will be deleted rt? but how come it is working for other repos but problem is having 5 or 6 repos

**Comment by sravula84** [comment]
I tried with that option but still failing with same error 

Post job cleanup.
/usr/local/bin/docker exec  ab4ad6a1cad0830443cd35398b8f1c41[2](https://github.com/xxxx/svc-experiment-tracking/actions/runs/4389931710/jobs/7687978229#step:22:2)1ecaf20[3](https://github.com/xxxx/svc-ssx-tracking/actions/runs/4389931710/jobs/7687978229#step:22:3)4b9eb709678d41ea7ec2ab9 sh -c "cat /etc/*release | grep ^ID"
Error relocating /__e/node16_alpine/bin/node: getentropy: symbol not found

**Comment by sethvargo** [comment]
Hi @sravula84 - I don't know. As I noted above, there are many ways to configure self-hosted runners including the underlying operating system and the base image. That error is coming from nodejs, not our GitHub Action; we do not run any `docker exec` commands.
